packages/components/src/components/hive-navigation/graphql-conf-card.tsx

```diff
@@ -12,7 +12,7 @@ export function GraphQLConfCard({ image }: GraphQLConfCardProps) {
       href="https://www.youtube.com/playlist?list=PL43V96KpNj7OMvmfL0WFKP6LpoboM8Hde"
       className="group w-[358px]"
     >
-      <Image alt="" {...image} width={358} height={200} />
+      <Image alt="" src={image} width={358} height={200} />
```

Comment:
why this? if image is `StaticImageData`

Reply:
I don't love this, but this is actually the way. The `src` prop accepts StaticImageMetadata.

https://nextjs.org/docs/pages/api-reference/components/image#src

```
src: string | StaticImageMetadata | { default: StaticImageMetadata }
```

<img width="950" alt="image" src="https://github.com/user-attachments/assets/05e0caaa-f3e7-4521-9f5b-c4f61f640fb3">


If you spread the image as separate props, you're getting a warning about blurHeight and blurWidth being unaccepted HTML attributes from React.

Reply:
oh yeah, you are right, this is a correct way of using static images - `src={image}` 🤦‍♂️